docs/HOWTO_HIRE_SOMEONE_TO_INSTALL_SPOKE.md

```diff
@@ -0,0 +1,93 @@
+# Howto Hire Someone to Deploy Spoke
+
+## Deciding to use Spoke
+
+You should treat all tech infrastructure choices as 'build vs buy' decisions, and make the right choice for your organization.
+
+Reasons to run Spoke:
+1. You want to run a peer-to-peer textbanking system at scale and pay only infrastructure costs
+2. You have the tech capacity to do the technical deployment work to set up Spoke and maintain your instance over time. 
+3. You want to be able to control costs and system scaling on your own terms. Perhaps you have experienced scaling problems from a textbanking vendor at an inopportune time. You can choose to control this risk by spending developer time and money on hardware running your own system.
```

Comment:
Another reason/use case:

You don't want to sign a contract with a vendor, and may have a more short term need for peer to peer texting.

Reply:
++